Forms/Schema/FormSchema.php

```diff
@@ -67,10 +67,10 @@ public function getState(Form $form) {
 			$this->getFieldStates($form->Actions())
 		);
 
-		if($form->Message()) {
-			$state['messages'][] = [
```

Comment:
I guess it's left a little unclear, are both (`Form` and `FormField`) APIs changing to plural messages, or only `Form` messages?